tests/unit/App.spec.ts

```diff
@@ -345,52 +349,53 @@ describe.skip('Numbered company setup', () => {
     localVue.use(VueRouter)
     const router = mockRouter.mock()
     await router.push({ name: 'define-company', query: { id: 'T7654321' } })
-    wrapper = shallowMount(App, { localVue, store, router, vuetify, stubs: { Affix: true } })
+    // Still need store for sbc-common-components
+    wrapper = shallowMount(App, { localVue, router, vuetify, stubs: { Affix: true } })
 
     // wait for all queries to complete
     await flushPromises()
   })
 
   it('loads a draft filing into the store', () => {
     // Validate IA for numbered company
-    expect(store.state.stateModel.tombstone.entityType).toBe('BEN')
-    expect(store.state.stateModel.tombstone.filingId).toBe(54321)
+    expect(store.stateModel.tombstone.entityType).toBe('BEN')
+    expect(store.stateModel.tombstone.filingId).toBe(54321)
 
     // Validate no offices are loaded
-    expect(store.state.stateModel.officeAddresses).toBeDefined()
-    expect(store.state.stateModel.officeAddresses.recordsOffice).toBeUndefined()
+    expect(store.stateModel.officeAddresses).toBeDefined()
+    expect(store.stateModel.officeAddresses.recordsOffice).toBeUndefined()
 
     // Validate Contact Info
-    expect(store.state.stateModel.businessContact).toBeDefined()
+    expect(store.stateModel.businessContact).toBeDefined()
 
     // Validate Share Structure
-    expect(store.state.stateModel.shareStructureStep.shareClasses).toBeDefined()
+    expect(store.stateModel.shareStructureStep.shareClasses).toBeDefined()
   })
 
   it('does not load a name request into the store', () => {
     // All Name request specific fields should be empty
-    expect(store.state.stateModel.nameRequest.nrNumber).toEqual('')
-    expect(store.state.stateModel.tombstone.filingId).toBe(54321)
+    expect(store.stateModel.nameRequest.nrNumber).toEqual('')
+    expect(store.stateModel.tombstone.filingId).toBe(54321)
 
     // Validate no NR Details
-    expect(store.state.stateModel.nameRequest.details.approvedName).toBeUndefined()
-    expect(store.state.stateModel.nameRequest.details.status).toBeUndefined()
-    expect(store.state.stateModel.nameRequest.details.consentFlag).toBeUndefined()
-    expect(store.state.stateModel.nameRequest.details.expirationDate).toBeUndefined()
+    expect((store.stateModel.nameRequest.details as any).approvedName).toBeUndefined()
```

Comment:
Why is this needed -- is `details` typed now? (That would be awesome.)

Reply:
Reverted, not sure why it wouldn't pick up details before

Reply:
Build can't handle `interface | object` for some reason

Reply:
I changed my mind. What happens if you change the declarations to `| any` instead? (in src/interfaces/store-interfaces/state-interfaces/name-request-interface.ts)

Reply:
What happens if you remove `| object` from the 3 interfaces that use it? I tried locally and don't see any errors.

Reply:
Seems to work now